CONTRIBUTING.md

```diff
@@ -115,4 +115,12 @@ $ node_modules/.bin/gulp serve --server=<che_api_url>
 
 
 ### Che server a.k.a WS master
-To be provided soon.
+For now Che in Che flow is not completed.
+However one may build Che server inside Che.
+To do that, create workspace from [Che devfile](https://github.com/eclipse/che/blob/master/devfile.yaml).
+Open a terminal in `ws/dev` container and run the following command from Che root directory:
+```bash
+mvn clean install -Pnative -DskipIntegrationTests
```

Comment:
I believe that this command should be part of the devfile.
There is already one command for building Che without tests https://github.com/eclipse/che/blob/master/devfile.yaml#L18

Reply:
Also, I wonder why we have inconsistency in profiles...
Here is an integration profile that is disabled by default, and we have another `integration-tests` profile special for typescript dto plugin.
Maybe it makes sense to align them and disable by default. 
@skabashnyuk @benoitf WDYT?